ClientApp/src/app/components/result-home/result-home.component.html

```diff
@@ -33,20 +30,6 @@ <h1>
 
 </div>
 
-<ng-template #generator >
```

Comment:
This change is not synced with staging. Please sync changes and restore the pull request.